pkg/svcgen/providers.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/izumin5210/grapi/pkg/grapicmd"
 	"github.com/izumin5210/grapi/pkg/protoc"
 	"github.com/izumin5210/grapi/pkg/svcgen/params"
+	_ "github.com/izumin5210/grapi/pkg/svcgen/template"
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
a blank import should be only in a main or test package, or have a comment justifying it (golint)